src/AdminDashboard/Fleets/PoliciesAdd.js

```diff
@@ -84,14 +85,13 @@ export default class FleetsAddPolicies extends Component {
     }
 
     render() {
-        console.log(this.state.selectedPolicies)
```

Comment:
do not forget to remove the `console.log ()`

Reply:
Perfect